### DISCUSSION TITLE
How to block user login if status not active, because i create user with active 0 still can login to system

### DISCUSSION THREAD

**Discussion by blipur** [codeigniter4/shield#1010] [Q&A] [answered]
I try create user with status 0 not active, but i try to login, the shield not block that user.

**Comment by kenjis** [comment]
See http://shield.codeigniter.com/references/authorization/#user-activation

**Reply by blipur** [reply]
But i'm use Shield default login controller in vendor, how i can modified that for check user id active or not. Because the builtin Shield not check the user active status.

**Reply by kenjis** [reply]
See http://shield.codeigniter.com/customization/extending_controllers/

**Reply by blipur** [reply, marked as answer]
thank's for your answer, i will learn that documentartion
